san/eventstudy.py

```diff
@@ -0,0 +1,277 @@
+import numpy as np
+import pandas as pd
+import seaborn as sns
```

Comment:
This dependency is missing in the requirements.txt

Reply:
It is not used at all. Removed.